### PR TITLE
Add Sugarchain Testnet support

### PR DESCRIPTION
**Pull request by ghost**
and fix slip44 patch for mainnet

### CODE REVIEW

File: configs/coins/sugarchain.json [hunk 1]
```diff
@@ -58,7 +58,7 @@
       "xpub_magic": 76067358,
       "xpub_magic_segwit_p2sh": 77429938,
       "xpub_magic_segwit_native": 78792518,
-      "slip44": 25252,
```
Comment (cryptozeny) [inline configs/coins/sugarchain.json:61]: i guess it was bellcoin? just fyi @ilmango-doge

File: configs/coins/sugarchain_testnet.json [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    "exclude_files": [
+      "bin/sugarchain-qt"
+    ],
+    "exec_command_template": "{{.Env.BackendInstallPath}}/{{.Coin.Alias}}/bin/sugarchaind -datadir={{.Env.BackendDataPath}}/{{.Coin.Alias}}/backend -conf={{.Env.BackendInstallPath}}/{{.Coin.Alias}}/{{.Coin.Alias}}.conf -pid=/run/{{.Coin.Alias}}/{{.Coin.Alias}}.pid",
+    "logrotate_files_template": "{{.Env.BackendDataPath}}/{{.Coin.Alias}}/backend/testnet5/*.log",
+    "postinst_script_template": "",
+    "service_type": "forking",
+    "service_additional_params_template": "",
+    "protect_memory": true,
+    "mainnet": false,
```
Comment (ghost) [inline configs/coins/sugarchain_testnet.json:39]: Rebased to fix this one 😂

File: configs/coins/sugarchain_testnet.json [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    "rpc_user": "rpc",
+    "rpc_pass": "rpc",
+    "rpc_timeout": 25,
+    "message_queue_binding_template": "tcp://127.0.0.1:{{.Ports.BackendMessageQueue}}"
+  },
+  "backend": {
+    "package_name": "backend-sugarchain-testnet",
+    "package_revision": "satoshilabs-1",
+    "system_user": "sugarchain",
+    "version": "0.16.3.21rc1",
```
Comment (cryptozeny) [inline configs/coins/sugarchain_testnet.json]: moonlight is .30

Comment (ghost) [reply configs/coins/sugarchain_testnet.json]: Good catch, thank you!

**Pull request closed without merging**